### ISSUE TITLE
Carriage return doesn't work on the first line of the snippet

### ISSUE THREAD

**Issue by TwentyH** [xmichelo/Beeftext#324]
This is on Windows 10, Edge browser. Beef Text 7.2.

I have a plain text snippet expanding ",thx" looking like that:
"Thanks,
Vincent"

![Screenshot 2020-11-19 085121](https://user-images.githubusercontent.com/2087439/99636970-75f40080-2a44-11eb-998d-7724f9703035.png)

It works fine most of the time. When typing in Edge browser (new message in Outlook), it expands as:
"Thanks,Vincent" => the carriage return is removed

![Screenshot 2020-11-19 085444](https://user-images.githubusercontent.com/2087439/99637326-f9155680-2a44-11eb-8da5-ef184442951a.png)

However if I change the snippet to:
"
Thanks,
Vincent"

![Screenshot 2020-11-19 085010](https://user-images.githubusercontent.com/2087439/99636834-49d87f80-2a44-11eb-94ed-2fca7d5783ed.png)
Then it correctly expand to:
"
Thanks,
Vincent"

![Screenshot 2020-11-19 085325](https://user-images.githubusercontent.com/2087439/99637148-b9e70580-2a44-11eb-8c20-55c5d2043974.png)

I think this is a bug.

**Comment by xmichelo** [comment]
Hi Vincent.

I was able to reproduce the issue, with Edge and Firefox. Unfortunately there is not much I can do. I tried other web mail applications (fastmail, gmail, Yahoo Mail), and the snippet works fine. Basically, what Beeftext is doing:

- Backup the content of the clipboard.
- Put the content of the snippet in the clipboard.
- Send a 'Paste' command.
- Restore the original content of the clipboard.

It's the Outlook editor that is messing with the formatting of the text, and there is nothing I can do about that.
